app/uk/gov/hmrc/accessibilitystatementfrontend/config/AppConfig.scala

```diff
@@ -23,7 +23,9 @@ import scala.io.Source
 
 @Singleton
 case class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
-  val contactHmrcUnauthenticatedLink = s"${servicesConfig.baseUrl("contact-frontend")}/contact-hmrc-unauthenticated"
+
+  private val contactHost = config.get[String]("contact-frontend.host")
+  val contactHmrcUnauthenticatedLink = s"$contactHost/contact/contact-hmrc-unauthenticated"
```

Comment:
I think this actually needs to be s"$contactHost/contact/accessibility" - like the link on https://www.tax.service.gov.uk/pay/accessibility-statement

Reply:
Good spot, fixed, thank you.